include/onnxruntime/core/common/logging/capture.h

```diff
@@ -82,6 +82,13 @@ class Capture {
     }
   }
 
+  char SysLogLevel() const noexcept {
+    // Carefully setup so severity_ is a valid index
+    GSL_SUPPRESS(bounds .2) {
+      return logging::SYSLOG_LEVEL[static_cast<int>(severity_)] - '0';
+    }
+  }
+
```

Comment:
As this is very syslog specific I think it should be in syslog_sink instead of here. You can call Capture::Severity() get the severity value there and do this mapping.

Reply:
This comment was marked resolved without indicating why it was resolved. What was the resolution?